test-site/package-lock.json

```diff
@@ -1739,16 +1739,51 @@
         "has-flag": "^4.0.0"
```

Comment:
does the top level package.json needs to be updated as well? since some of these seems to be linked to the top level node_modules

Reply:
only `terser` (and its dependencies) is in the top-level package.json too and it is already on a vulnerability-free version. we might have updated it there and then forgotten to also update it in the test-site